recipe/meta.yaml

```
@@ -114,7 +109,7 @@ outputs:
   - name: ray-default
     requirements:
       host:
-        - python
+        - python <3.9
```

```suggestion
        - python
```

In general we don't want constrains attached to packages already reflected in the pinning (check your yaml files under `.ci_support`).